src/models/StyledComponent.js

```diff
@@ -178,10 +178,10 @@ export default (ComponentStyle: Function, constructWithOptions: Function) => {
         return
       }
 
-      this.setState(oldState => {
+      this.setState(prevState => {
```

Comment:
the react eslint plugin is kind of dumb and requires you to name stuff like this in a certain way to be detected